### PR TITLE
feat: add support for new fetchedAt property

### PR DESCRIPTION
**Pull request by LeoPaulusSmartCar**
Added support for new property and updated corresponding tests

### CODE REVIEW

**Comment by LeoPaulusSmartCar** [comment]
I see the builds are failing, but they also failed for previous PRs. Also, there are a few tests that are failing in master that might be causing the issue

File: test/unit/lib/vehicle.js [hunk 1]
```diff
@@ -82,36 +82,54 @@ test('vehicle webhook subscribe', async(t) => {
...
+      'sc-fetched-at': '2018-05-04T07:20:51.844Z',
+    });
 
   const response = await vehicle.subscribe('webhookID');
 
   t.is(response.webhookId, 'webhookID');
   t.is(response.vehicleId, 'vehicleId');
   t.is(response.meta.requestId, 'requestId');
+  t.is(response.meta.dataAge.valueOf(), 1525418450844);
+  t.is(response.meta.fetchedAt.valueOf(), 1525418451844);
```
[the remark this answers is not in the document]
Comment (LeoPaulusSmartCar) [reply test/unit/lib/vehicle.js:97]: I think all requests go through the same pipeline that includes the headers, so I think it is a good practice to check for them

File: doc/readme.md
Comment (alejandrosmartcar) [inline doc/readme.md:1]: I have checked the project and the documentation files are automatically generated when running:
- npm run jsdoc
- npm run docs
To do this you need to correctly complete the jsdoc for `lib/vehicle.js`

File: lib/vehicle.js
Comment (alejandrosmartcar) [inline lib/vehicle.js:1]: Search for the word `dataAge` and you will find jsdoc format comments where I think the `fetchedAt` should be added so that when running the documentation commands it automatically adds them.
For example lines 277, 521

File: test/unit/lib/vehicle.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (alejandrosmartcar) [reply test/unit/lib/vehicle.js:97]: According the `api` project, the `fetchedAt` are added in `extractMetaValues`.
In general, the `extractMetaValues` is used in the `format` step of the `processors` that bring signals (e.g.: 'GET /odometer', 'POST /batch', 'GET /:make(tesla)/speedometer'), where `processors/vehicle/index.js` calls `processors/vehicle/formatters/core.js` to apply the corresponding `format` for each signal.

However, in the `format` of the `webhook` endpoints I don't see that they call `extractMetaValues` (Ex: `processors/vehicle/subscribe-vehicle-to-webhook.js`)

In the `request - override non-sc headers` test, the objective of the test seems to be another, simply to clean the headers sent by the user that do not begin with `sc-` so as not to return them.

File: test/unit/lib/vehicle-coverage.js
Comment (alejandrosmartcar) [inline test/unit/lib/vehicle-coverage.js:1]: I don't know what the need was for creating the `vehicle-coverage.js` file. If they are tests for `vehicle` requests they could have been inside the existing `vehicle.js` file.

**Comment by alejandrosmartcar** [comment]
The comments I posted are what I appreciate about the project, but this is the first time I've seen this project, I don't know if someone else can review it.

File: lib/vehicle.js
Comment (LeoPaulusSmartCar) [reply lib/vehicle.js:1]: added!

File: test/unit/lib/vehicle-coverage.js
Comment (LeoPaulusSmartCar) [reply test/unit/lib/vehicle-coverage.js:1]: This were added because one of we were not reaching the 95% code coverage required, and I used a separate file for them because vehicle.js have the main tests for user-facing API functionality, meanwhile I vehicle-coverage.js tests more internal methods, edge cases, errors, etc. Also, it was created for the purpose of increasing the coverage, therefore the name.

Comment (alejandrosmartcar) [reply test/unit/lib/vehicle-coverage.js:1]: From another point of view the lib folder has: `auth-client.js`, `utils.js` and `vehicle.js`.

So it seems to me that the tests in `test/end-to-end` and `test/unit/lib` should keep the same structure and there should be the files `auth-client.js`, `utils.js` and `vehicle.js`.

If it is to add coverage of the vehicle.js file I would suggest adding it in `end-to-end/vehicle.js` or `unit/lib/vehicle.js`.

File: doc/readme.md
Comment (alejandrosmartcar) [reply doc/readme.md:1]: Did you run both?
I don't see that `doc/readme.md` has been updated.

File: lib/vehicle.js [hunk 2]
```diff
@@ -705,9 +712,11 @@ Vehicle.prototype.request = async function(
 /**
  * @type {Object}
  * @typedef NominalCapacity
- * @property {Array} availableCapacities - A list of the rated nominal capacities available for a vehicle
+ * @property {Array} availableCapacities - A list of the rated nominal
```
Comment (alejandrosmartcar) [inline lib/vehicle.js:715]: eslint is throwing error because there is a space at the end of the line

File: lib/vehicle.js [hunk 3]
```diff
@@ -705,9 +712,11 @@ Vehicle.prototype.request = async function(
 /**
  * @type {Object}
  * @typedef NominalCapacity
- * @property {Array} availableCapacities - A list of the rated nominal capacities available for a vehicle
+ * @property {Array} availableCapacities - A list of the rated nominal 
+ * capacities available for a vehicle
  * @property {Object} capacity - The rated nominal capacity for the vehicle's battery in kWh
- * @property {String} url - A URL that will launch the flow for a vehicle owner to specify the correct battery capacity for a vehicle
+ * @property {String} url - A URL that will launch the flow for a vehicle owner
```
Comment (alejandrosmartcar) [inline lib/vehicle.js:718]: eslint is throwing error because there is a space at the end of the line

File: doc/readme.md
Comment (alejandrosmartcar) [inline doc/readme.md:1]: Please re-run `npm run docs`

Travis CI is failing when `test -z "$(git diff --name-only | grep '^doc/readme.md$')"`
`The command "test -z "$(git diff --name-only | grep '^doc/readme.md$')"" exited with 1.`

**Review by alejandrosmartcar — CHANGES_REQUESTED** [review]
